src/scenes/Projects/Overview/ProjectOverview.tsx

```diff
@@ -0,0 +1,154 @@
+import {
+  Breadcrumbs,
+  Chip,
+  fade,
+  IconButton,
+  makeStyles,
+  Typography,
+} from '@material-ui/core';
+import { Add, Publish } from '@material-ui/icons';
+import { FC } from 'react';
+import * as React from 'react';
+import { useLocation } from 'react-router-dom';
+import {
+  InternshipProject,
+  LanguageEngagement,
+  TranslationProject,
+} from '../../../api/schema.generated';
+import { Breadcrumb } from '../../../components/Breadcrumb';
+import { BudgetOverviewCard } from '../../../components/BudgetOverviewCard';
+import { CardGroup } from '../../../components/CardGroup';
+import { LanguageEngagementListItemCard } from '../../../components/LanguageEngagementListItemCard';
+import { PartnershipSummary } from '../../../components/PartnershipSummary';
+import { ProjectMembersSummary } from '../../../components/ProjectMembersSummary';
+import { useProjectOverviewQuery } from './ProjectOverview.generated';
+
+const useStyles = makeStyles(({ spacing, palette }) => ({
+  root: {
+    flex: 1,
+    overflowY: 'scroll',
+    padding: spacing(4),
+    '& > *': {
+      marginBottom: spacing(3),
+    },
+  },
+  container: {
+    display: 'flex',
+    alignItems: 'center',
+  },
+  uploadInput: {
+    display: 'none',
+  },
+  uploadButton: {
+    backgroundColor: palette.primary.main,
+    color: palette.common.white,
+    marginRight: spacing(1),
+    '&:hover': {
+      backgroundColor: fade(palette.primary.main, 0.5),
+    },
+  },
+  chips: {
+    '& > *': {
+      marginRight: spacing(2),
+    },
+  },
+  budgetOverviewCard: {
+    marginRight: spacing(3),
+  },
+  addButton: {
+    marginLeft: 'auto',
+    marginRight: spacing(1),
+    backgroundColor: palette.error.main,
+    color: palette.common.white,
+    '&:hover': {
+      backgroundColor: fade(palette.error.main, 0.5),
+    },
+  },
+}));
+
+export const ProjectOverview: FC = () => {
+  const classes = useStyles();
+  const { data, error } = useProjectOverviewQuery();
```

Comment:
Missing pulling id from path and passing in here.

Reply:
Woops... I'm passing this in now. Getting an error from the request about some fields not existing but it looks api related

Reply:
![Screen Shot 2020-06-04 at 7 46 30 PM](https://user-images.githubusercontent.com/16201433/83821477-766a2c00-a69c-11ea-806b-b322e27d13c6.png)